.env.development

```diff
@@ -1,13 +1,14 @@
 REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
 REACT_APP_API_ROOT=https://dev-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://dev-k8s.treetracker.org/earnings
+REACT_APP_FIELD_DATA_ROOT=http://localhost:3003
```

Comment:
Try not to include `localhost` APIs in .env.development. Instead, put them in `.env.local`